nebu/cli/get.py

```diff
@@ -26,18 +29,40 @@ def get(ctx, env, col_id, col_version, output_dir, book_tree):
 
     base_url = build_archive_url(ctx, env)
 
+    version = None
+    req_version = col_version
+    if col_version.count('.') > 1:
+        full_version = col_version.split('.')
+        col_version = '.'.join(full_version[:2])
+        version = '.'.join(full_version[1:])
+
     col_hash = '{}/{}'.format(col_id, col_version)
     # Fetch metadata
     url = '{}/content/{}'.format(base_url, col_hash)
     resp = requests.get(url)
     if resp.status_code >= 400:
-        raise MissingContent(col_id, col_version)
+        raise MissingContent(col_id, req_version)
     col_metadata = resp.json()
     if col_metadata['collated']:
         url = resp.url + '?as_collated=False'
         resp = requests.get(url)
+        if resp.status_code >= 400:
+            # This should never happen - indicates that only baked exists?
+            raise MissingContent(col_id, req_version)
```

Comment:
Added this test case since we should _never_ assume requests succeed, if we're never going to show a backtrace to the user. BTW, in making the test-coverage work for this, discovered that we've been mocking archive wrong, a bit - we should explicitly put in the 301 redirects from legacy to uuid urls, like I did in the last test. Don't _quite_ have the gumption to fix all the other tests, yet. (Seems like something to refactor out, maybe)